docs/source/examples/managed-jobs.rst

```diff
@@ -625,6 +625,28 @@ you can still tear it down manually with
 .. note::
   Tearing down the jobs controller loses all logs and status information for the finished managed jobs. It is only allowed when there are no in-progress managed jobs to ensure no resource leakage.
 
+To adjust the size of the jobs controller instance, see :ref:`jobs-controller-custom-resources`.
+
+
+Setup and best practices
+------------------------
+
+.. _static-creds:
+
+Using static credentials
```

Comment:
Q: is "static credentials" the right term? I've always associated with a static file, e.g., ~/.aws/credentials. But what we referred to below = service accounts?

Do both static-file and service-accounts work here?

Reply:
Here's my intended meaning:
- "static credentials" - a credential file that does not need to be refreshed/expire, like ~/.aws/credentials
- "service account" - the user role/account used by a service. In GCP, you can create a static credential that authenticates a service account. In AWS, there's no exact service account concept - you can create instance profiles to assume roles, or create IAM users with attached roles which could use static credentials.

Basically "service account" is just kind of an overloaded term. In GCP, the thing you have to create is literally called "service account" in the console, so that's why it is used below.

Reply:
Maybe add the specific AWS and GCP examples as a short sentence here will help? E.g.,  ~/.aws/credentials on AWS or a GCP service account that uses GOOGLE_xxx_env_var=yyy.json?

Reply:
Still thinking "Using static credentials" is potentially confusing. Wdyt about "Using long-lived credentials"?

Reply:
Do you think "Using long-lived static credentials" is too much?

Reply:
Reason I wanted to avoid "static credentials" is because it's not a generic term across clouds. "long-lived" or "non-expirable" is more generic.

Reply:
Do you think "Using long-lived static credentials" is too much?

Reply:
Do you think "Using long-lived static credentials" is too much?